test/unit/os/shell_spec.lua

```diff
@@ -81,19 +81,19 @@ describe('shell functions', function()
 
   describe('shell_build_argv', function()
     itp('works with NULL arguments', function()
-      eq({'/bin/bash'}, shell_build_argv(nil, nil))
+      eq({'/bin/sh'}, shell_build_argv(nil, nil))
```

Comment:
Yes.  They failed because the argument vector is built from the string specified earlier in the test, and changed as that string changed.